packages/stratocacher-layer-rls/test/index.js

```diff
@@ -0,0 +1,24 @@
+const mockRequire = require("mock-require");
+
+var CACHE = {};
+
+mockRequire("request-local-storage", {
+	getNamespace: () => CACHE,
+});
```

Comment:
Why not just use real `request-local-storage` for the test?

Reply:
Good call!